CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## dev
 
-Version <dev> updates View Componment instrumentation to use a default metric name when one is unavailable, adds a configuration option to associate the AWS account ID with the DynamoDB calls from the AWS SDK, resolves a bug in rdkafka instrumentation when using the karafka-rdkafka gem, resolves a bug in the ruby-kafka instrumentation, and fixes a bug with Grape instrumentation.
+Version <dev> updates View Componment instrumentation to use a default metric name when one is unavailable, adds a configuration option to associate the AWS account ID with the DynamoDB calls from the AWS SDK, resolves a bug in rdkafka instrumentation when using the karafka-rdkafka gem, resolves a bug in the ruby-kafka instrumentation, fixes a bug with Grape instrumentation, and addresses a bug preventing Ruby based AWS Lambda layer contexts from working properly.
```

Comment:
```suggestion
Version <dev> updates View Componment instrumentation to use a default metric name when one is unavailable, adds a configuration option to associate the AWS account ID with the DynamoDB calls from the AWS SDK, resolves a bug in rdkafka instrumentation when using the karafka-rdkafka gem, resolves a bug in the ruby-kafka instrumentation, fixes a bug with Grape instrumentation, and addresses a bug preventing the agent from installing in an AWS Lambda layer.
```

Reply:
@kaylareopelle with v9.14.0 the agent is installed and just doesn't realize it's meant to be operating in serverless mode. So I've been trying to go with "operating", "functioning", etc. type phrasing. Any ideas for improving the summary text based on that clarification?

Reply:
Ah, thanks for clarifying! Maybe "preventing the agent from running in serverless mode in an AWS Lambda layer"? (could also use operating or functioning... I like that operating is used in the description below)

Reply:
Cool. Co-authored [8a8bd45](https://github.com/newrelic/newrelic-ruby-agent/pull/2920/commits/8a8bd454cfb135f5cd8546d215aee58ce1f3482a).